saigon-tech/laravel-exercise
# #1 Migration & Seeder

### Requests

- Migration and seeder.

### Changes

- Migrate table in database ([document](https://docs.google.com/spreadsheets/d/12Z4jH4arqmLzo232QT618ao0Gc-n-aSPwBXnwTMW46o/edit#gid=0))
- Create seeder for tables.

**Testing**

- Connect container php: docker-compose exec php bash.
- Run command: php artisan migrate --seed.
- Check result in database.
